### DISCUSSION TITLE
Best Proxy and Setup for Security Onion

### DISCUSSION THREAD

**Discussion by veraxcyber** [Security-Onion-Solutions/securityonion#5442] [Unsupported Versions] [answered]
Hello all, 

I currently have my Security Onion sensor setup with a SPAN port and the sensor is a dedicated host. Is it possible to setup a proxy on the Security Onion sensor to monitor web traffic or does it need to be a separate host? 

Which proxy would you recommend using please? 

Many Thanks!

**Comment by dougburks** [comment]
I'm not sure that I understand what you're asking. If Security Onion is already monitoring the traffic from a SPAN port, then it should already be logging web traffic that flows across that SPAN port without having to use a proxy.

**Comment by veraxcyber** [comment]
Hello,

Thanks for the response,

I am looking to have a proxy to block traffic to malicious domains and wanted to know if this could be setup on the security onion host?

Many Thanks

Andrew

**Comment by dougburks** [comment, marked as answer]
You may be able to make this work, but we don't recommend or support it. Most organizations run their proxy on a separate box.

**Comment by veraxcyber** [comment]
Hello Doug,

Appreciate the response and I think this seems to be the advice of most and thanks again for your help.

Andrew
